core/storage/story/gae_models.py

```diff
@@ -77,22 +77,21 @@ def _trusted_commit(
                     cmd: str. Unique command.
                 and then additional arguments for that command.
         """
-        super(StoryModel, self)._trusted_commit(
-            committer_id, commit_type, commit_message, commit_cmds)
-
         committer_user_settings_model = (
             user_models.UserSettingsModel.get_by_id(committer_id))
         committer_username = (
             committer_user_settings_model.username
             if committer_user_settings_model else '')
 
         story_commit_log_entry = StoryCommitLogEntryModel.create(
-            self.id, self.version, committer_id, committer_username,
+            self.id, self.version + 1, committer_id, committer_username,
```

Comment:
Can you explain why this was changed? Just curious, and wondering whether we need to do it elsewhere too.

Reply:
(explanation above) We might need to, I didn't test with explorations and collections yet. Will change it if it also has problems.

Reply:
I checked and the existing code seems to work for collections and explorations.

Reply:
Done, changed it to put() as discussed.